taxonomy-forum.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * taxonomy-forum.php
+ *
+ * This is the default template for Forums (forum taxonomy).
+ *
+ * To modify this template:
+ * - Create a groups-forums subfolder in your theme's root folder.
+ * - Copy this file there and adjust it as desired.
+ *
+ * @author itthinx
+ * @package groups-forums
+ * @since groups-forums 1.0.0
+ */
+
+get_header();
+
+echo '<div id="primary" class="site-content forum">';
+echo '<div id="content">';
+
+// title & New Topic link.
+if (is_tax()) {
```

Comment:
Please review code formatting here and elsewhere, e.g. this line should be:

`if ( is_tax() ) {`